src/main/java/gov/usgs/owi/nldi/controllers/NetworkController.java

```diff
@@ -79,4 +88,48 @@ public void getFeatures(HttpServletRequest request, HttpServletResponse response
 		}
 	}
 
+
+	//swagger documentation for /linked-data/comid/position endpoint
+	@Operation(summary = "getFeatureByCoordinates", description = "returns the feature closest to a provided set of coordinates")
+	@ApiResponses(value = {
+		@ApiResponse(responseCode = "200", description = "OK", content = { @Content(mediaType = "application/json",
+			schema = @Schema(implementation = Feature.class)) }),
+		@ApiResponse(responseCode = "500", description = "Server error", content = @Content) })
+	@GetMapping(value="linked-data/comid/position", produces=MediaType.APPLICATION_JSON_VALUE)
+	public void getFeatureByCoordinates(
+		HttpServletRequest request, HttpServletResponse response,
+		@Parameter(description=Parameters.COORDS_DESCRIPTION)
+		@Pattern(message=Parameters.POINT_VALIDATION_MESSAGE, regexp=Parameters.POINT_VALIDATION_REGEX)
+		@RequestParam(value=Parameters.COORDS) String coords) throws Exception {
+		BigInteger logId = logService.logRequest(request);
+		Map<String, Object> map = extractLatitudeAndLongitude(coords);
+		try {
+        	Integer comid = lookupDao.getComidByLatitudeAndLongitude(map);
+			FeatureTransformer transformer = new FeatureTransformer(response, configurationService);
+
+			Map<String, Object> parameterMap = new HashMap<> ();
+			parameterMap.put(LookupDao.FEATURE_SOURCE, parameters.COMID);
+			parameterMap.put(Parameters.FEATURE_ID, comid);
+			addContentHeader(response);
+			streamResults(transformer, BaseDao.FEATURE, parameterMap);
+		} catch (Exception e) {
+			GlobalDefaultExceptionHandler.handleError(e, response);
+		} finally {
+			logService.logRequestComplete(logId, response.getStatus());
+		}
+	}
+
+	private Map<String, Object> extractLatitudeAndLongitude(String coords) throws UnsupportedEncodingException {
```

Comment:
Seems like this method is not calling anything declared to throw UnsupportedEncodingException
So can be removed.